test/fuzz/meson.build

```diff
@@ -15,7 +15,7 @@ sanitize_address_undefined = custom_target(
                    project_source_root,
                    '@OUTPUT@',
                    'fuzzers',
-                   '-Db_lundef=false -Db_sanitize=address,undefined @0@'.format(optimization),
+                   '-Dfuzz-tests=true -Db_lundef=false -Db_sanitize=address,undefined @0@'.format(optimization),
```

Comment:
Also, I have no idea why it works without `-Dslow-tests=true` or `-Dfuzz-tests=true` before.